it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
+12564=L'autore di questo messaggio ha richiesto che le risposte siano inviate solamente all'autore. Se anche tu vuoi rispondere nel gruppo di discussione, aggiungi un nuova riga all'area dell'indirizzo, scegli Newsgroup dalla lista dei destinatari ed inserisci il nome del newsgroup.
 
 ## @name NS_MSG_UNDISCLOSED_RECIPIENTS
 ## LOCALIZATION NOTE: this string must be using only US_ASCII characters
-12566=destinatari-ignoti
+12566=destinatari-non-rivelati
 
 ## @name NS_MSG_ERROR_ATTACHING_FILE
-12570=Errore nell'allegare %S. Controllare i permessi di accesso al file.
+12570=Errore nell'allegare %S. Verifica se hai accesso al file.
 
 ## @name NS_MSG_ERROR_DOING_FCC
-12571=Errore durante la copia del messaggio nella cartella 'Posta inviata'. Riprovare?
+12571=Si è verificato un errore durante la copia del messaggio nella cartella 'Posta inviata'. Riprovare?
 
 ## @name NS_ERROR_SMTP_GREETING
-12572=Si è verificato un errore durante l'invio della posta: il server di posta ha inviato una stringa non corretta %s.
+12572=Si è verificato un errore durante l'invio della e-mail: il server di posta ha inviato una risposta non corretta %s.
 
 ## @name NS_ERROR_SENDING_RCPT_COMMAND
-12575=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %1$s. Controllare il destinatario del messaggio %2$s e riprovare.
+12575=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %1$s. Verifica il destinatario del messaggio %2$s e riprova.
 
 ## @name NS_ERROR_COULD_NOT_LOGIN_TO_SMTP_SERVER_AUTH_NONE
-12581=Errore durante l'invio della posta: impossibile autenticarsi presso il server SMTP %S. Il server non consente alcun meccanismo di autenticazione (SMTP-AUTH). Togliere la spunta da "Usa nome utente e password" per quel server o contattare il proprio service provider.
+12581=Si è verificato un errore durante l'invio della e-mail: impossibile autenticarsi presso il server SMTP %S. Il server non supporta l'autenticazione (SMTP-AUTH), ma hai scelto ugualmente di utilizzarla. Togli la spunta a "Usa nome utente e password" per il server o contatta il tuo fornitore di servizi.
 
 ## @name NS_ERROR_STARTTLS_FAILED_EHLO_STARTTLS
-12582=Errore durante l'invio della posta: impossibile stabilire un collegamento sicuro col server SMTP %S usando STARTTLS in quanto non supporta tale caratteristica. Disattivare STARTTLS per quel server o contattare il proprio service provider.
+12582=Si è verificato un errore durante l'invio della e-mail: impossibile stabilire un collegamento sicuro col server SMTP %S utilizzando STARTTLS in quanto non supporta tale caratteristica. Disattiva STARTTLS per il server o contatta il tuo fornitore di servizi.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12582=Si è verificato un errore durante l'invio dell'e-mail: impossibile stabilire un collegamento sicuro col server SMTP %S utilizzando STARTTLS in quanto non supporta tale caratteristica. Disattiva STARTTLS per il server o contatta il tuo fornitore di servizi.